tmpl/preamble.md

```diff
@@ -0,0 +1,10 @@
+## Instrumented modules
+
+After installation, the agent automatically instruments with our catalog of
+supported Node.js libraries and frameworks. This gives you immediate access to
+granular information specific to your web apps and servers.  For unsupported
+frameworks or libraries, you'll need to instrument the agent yourself using the
+[Node.js agent API](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/).
```

Comment:
I noticed this before but should we just link out to https://newrelic.github.io/node-newrelic/API.html?

Reply:
Done.